### DISCUSSION TITLE
insar_vs_GPS comparison with geometry in geographical coordinates

### DISCUSSION THREAD

**Discussion by amrit-thapa-2044** [insarlab/MintPy#1010] [Q&A]
Hello! I'm wondering if anyone has attempted to modify the code available at [https://github.com/insarlab/MintPy/blob/main/src/mintpy/objects/insar_vs_gps.py ] to support geometry in a geographical coordinate system [Hyp3-MintPy workflow]. It looks like the current implementation of the script relies on radar coordinates.

**Comment by scottstanie** [comment]
The closest I have seen is here: https://github.com/nisar-solid/ATBD/blob/dcbf4229443dadf0eee0863144fbec852654fb56/methods/secular/Secular_Requirement_Validation.ipynb (Section 4.2 - 4.5)
They are doing a GPS vs InSAR comparison on geocoded (ARIA) products, but they are using the `GPS` class instead of the `insar_vs_gps` object.

I'm not sure which is more common to use, as there's overlapping functionality in the two parts of the `gps.py` module. I'll update here if/when I get a version that's quick and easy to use
